discovery-provider/plugins/network-monitoring/src/content/index.ts

```diff
@@ -177,7 +186,7 @@ export const checkCID = async (
 ) => {
     console.log(`[${run_id}:${spid}] check cids`)
 
-    const saveQueue: Promise<void>[] = []
+    // const saveQueue: Promise<void>[] = []
```

Comment:
?

Reply:
This is just commenting out the old way of saving to the DB - this is for CIDs so really the function isn't even used yet,

Reply:
why keep old code around vs just deleting

Reply:
Because it's one of the quick fixes for limiting the number of DB connection - ideally I'd like to keep the `saveQueue` but it was one of the reasons the DB was getting overloaded and transactions were timing out

I don't mind deleting it for now though and just adding it back at a later date.